flow-renderer-parent/flow-renderer/src/test/java/com/vaadin/flow/renderer/tests/RendererSerializableTest.java

```diff
@@ -0,0 +1,15 @@
+package com.vaadin.flow.renderer.tests;
```

Comment:
These I needed to add to make CI pass. See https://github.com/vaadin/flow/issues/11371 Not related to this feature in any way.

Reply:
Resolved by https://github.com/vaadin/flow/issues/11371#issuecomment-872774648